src/core/thread/mesh_forwarder.cpp

```diff
@@ -1254,6 +1259,13 @@ ThreadError MeshForwarder::SendFragment(Message &aMessage, Mac::Frame &aFrame)
 
         if (payloadLength > fragmentLength)
         {
+            if (!aMessage.IsLinkSecurityEnabled())
```

Comment:
👍  I think having a way to determine if a message is MLE would be helpful later on (for example, when we  implement the re-claim buffer/message...)